StyleGuide.md

```diff
@@ -538,11 +536,9 @@ To insert a note, use the following code:
 {{site.data.alerts.end}}
 ~~~
 
-Each Liquid tag should be on its own line. You can use Markdown within the highlighted text.
-
 #### Warnings
 
-Use a warning to express that a piece of information is critical to understand to prevent unexpected things from happening.
+Use a warning to express that a piece of information is critical to understand to prevent data loss, security vulnerability, or unexpected behavior.
```

Comment:
```suggestion
Use a warning to express that a piece of information is critical to understand to prevent data loss, security vulnerabilities, or unexpected behavior.
```